src/Modules/AuthModule/components/ChangePassword/ChangePassword.tsx

```diff
@@ -1,6 +1,251 @@
+import { useForm } from "react-hook-form";
+import logo from "../../../../assets/images/PMS 3.svg";
+import { FormDataChangPass } from "../../../../interfaces/Auth";
+import "../Login/Login.css";
+import { useContext, useState } from "react";
+import { toast } from "react-toastify";
+import axios from "axios";
+import { AuthContext } from "../../../../context/AuthContext";
 
 export default function ChangePassword() {
+
+  const {
+    register,
+    handleSubmit,
+    formState: { errors },
+    watch,
+  } = useForm<FormDataChangPass>();
+
+  const { baseUrl } = useContext(AuthContext);
+
+  // Password eye
+  const [showPassword, setShowPassword] = useState<boolean>(false);
```

Comment:
I don't like the name `showPassword` as it indicates to the reader that it is a function, not a state variable.
We change it to something like `[isPasswordShown,setIsPasswordShown]`. The new name encapsulates that it is a value and a boolean one.